src/main.cpp

```diff
@@ -504,6 +505,12 @@ int main(const int argc, const char *const *const argv) {
     ldLog() << "Prepending QT_INSTALL_LIBS path to $LD_LIBRARY_PATH, new $LD_LIBRARY_PATH:" << newLibraryPath.str()
             << std::endl;
 
+    std::ostringstream newPath;
+    newPath << qtBinsPath.string() << ":" << getenv("PATH");
+    setenv("PATH", newPath.str().c_str(), true);
```

Comment:
That seems like a really, really bad hack, I also don't like the logging there (this should be debug level, at most). I'll check if we can do a bit more elegant when I'm home.

Reply:
Agreed, I just wanted to keep the same (ugly) pattern used for `QT_INSTALL_LIBS`

Reply:
Right, didn't see that. Okay, well, in that case, can you add an issue so we will refactor this at some point?

Let's merge this for now. Thanks for fixing this.